StrategyGame/Model/Heroes/Archer.cs

```diff
@@ -1,9 +1,38 @@
-﻿namespace StrategyGame.Model.Heroes
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using StrategyGame.Annotations;
+
+namespace StrategyGame.Model.Heroes
 {
     public class Archer : IHero
     {
-        public int Hitpoints { get; set; }
-        public int Damage { get; set; }
-        public int Speed { get; set; }
+        private int _hitpoints;
```

Comment:
Just may be useful:
http://stylecop.soyuz5.com/Ordering%20Rules.html
At least: SA1201, SA1202, SA1203;